packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,351 @@
+    {key: 'b'},
+    {key: 'c'},
+    {key: 'd'},
+    {key: 'e'},
+    {key: 'f'},
+    {key: 'g'},
+    {key: 'h'},
+    {key: 'i'},
+    {key: 'j'},
+    {key: 'k'},
+    {key: 'l'},
+    {key: 'm'},
+    {key: 'n'}
+];
+
+<ListView items={rows} height="250px" aria-label="List example">
+  {(item) => (
+    <Item key={item.key} textValue={`Item ${item.key}`}>
+      {`Item ${item.key}`}
+    </Item>
+  )}
+</ListView>
+```
+
+### Internationalization
+To internationalize a ListView, all text content within the ListView should be localized. This includes the `aria-label` provided to the ListView if any.
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of ListView is automatically flipped.
+
+## Labeling
+### Accessibility
+An `aria-label` must be provided to the ListView for accessibility. If the ListView is labeled by a separate element, an `aria-labelledby` prop must be provided using the id of the labeling element instead.
+
+## Asynchronous loading
+ListView supports loading data asynchronously, and will display a progress circle reflecting the current load state,
+set by the `loadingState` prop. It also supports infinite scrolling to load more data on demand as the user scrolls, via the `onLoadMore` prop.
+
+This example uses the [useAsyncList](../react-stately/useAsyncList.html) hook to handle loading the data.
+See the docs for more information.
+
+```tsx example
+import {useAsyncList} from '@react-stately/data';
+
+function AsyncList() {
+
+  let list = useAsyncList({
+    async load({signal, cursor}) {
+      if (cursor) {
+        cursor = cursor.replace(/^http:\/\//i, 'https://');
+      }
+
+      let res = await fetch(cursor || `https://swapi.py4e.com/api/people/?search=`, {signal});
+      let json = await res.json();
+
+      return {
+        items: json.results,
+        cursor: json.next
+      };
+    }
+  });
+
+  return (
+    <ListView
+        aria-label="example async loading list"
+        height="size-3000"
+        items={list.items}
+        loadingState={list.loadingState}
+        onLoadMore={list.loadMore}>
+        {(item) => (
+          <Item key={item.name} textValue={item.name}>{item.name}</Item>
+        )}
+    </ListView>
+  );
+}
+```
+
+## Complex items
+
+Items within a ListView also allow for additional content used to add context or provide additional actions to Items. Icons, links, and descriptions can be added to the children of Item as shown in the example below. If a description is added, the prop `slot="description"` must be used to distinguish the different `<Text>` elements. Additionally, components such as `<ActionButton>`, `<ActionGroup>`, and `<ActionMenu>` will be styled appropriately if included within an item.
+
+```tsx example
+<ListView aria-label="List example with complex items">
+  <Item textValue="Row with link" hasChildItems>
+    <Link>Row with link</Link>
+  </Item>
+  <Item textValue="Row with link, icon, and description" hasChildItems>
```

Comment:
Maybe we wanna omit the `hasChildItems` bit for now, not sure if we completely support that yet

Reply:
Removed for now